### DISCUSSION TITLE
Firefox

### DISCUSSION THREAD

**Discussion by Willxiam** [pw-64/TabCollector#2] [Ideas]
This is great.  I would love the same plugin for firefox.

**Comment by pw-64** [comment]
Thank you for the support. It is currently in development; however, I am having issues porting it to Firefox. I will commit (and mention here) in a moment with the current plugin code.

**Reply by pw-64** [reply]
See commit 44ef75b8524684d8ad96897d1e906bd32011a9dc

**Reply by pw-64** [reply]
The current issue is JavaScript apparently not running, since no confirmation popup appears upon click.

**Reply by pw-64** [reply]
Clone the repo and `cd TabCollector` and use FF's [`web-ext`](https://github.com/mozilla/web-ext). This method is convenient when changing source code as there is a live-reload, so I would recommend using that over the manual packing method using the "Pack extension" feature in chrome://extensions.

**Comment by pw-64** [comment]
@Willxiam  With the release of v1.1 comes the first release for Firefox. This is just a copy of the Chrome extension at the moment, so there may be issues. The next release will be properly ported and any Firefox-specific issues should be fixed - if you find any please open an issue with the `[BUG]` tag.

**Comment by pw-64** [comment]
Firefox should be properly ported now.

**Comment by Willxiam** [comment]
@pw-64 Thank you.  Sorry I have not replied life has been busy. I am going to try it out this weekend.

**Reply by pw-64** [reply]
No worries, thank you for the interest in the project.
